### DISCUSSION TITLE
What is the purpose of setting up database replication?

### DISCUSSION THREAD

**Discussion by karozans** [eldy/SellYourSaas#175] [Q&A] [answered]
In the directions for setting up SYS, you create replication between the mysql dbases on the Admin and the Deployment servers in a Master-Slave relationship.  This means that only the dbases on the Admin servers are replicated down to the Deployment servers.  The Deployment servers will never replicate the new dbase instances up to the Admin servers.

When you create a new instance on the Deployment servers, the database is create on the Deployment servers, but the Master-Slave relationship does not replicate the database up to the Admin server.  Also on the deployment servers, they are setup to ignore the dolibarr dbase on the Admin servers and a few others.

So it doesn't appear that anything is getting replicated.  So what is the purpose of setting up replication?

Are the installation instructions wrong?

**Comment by eldy** [comment, marked as answer]
Goal was having duplication of the master database. But it has been abandonned (it is flagged in doc as optional, but clearly useless).
